doc/sphinx/source/recipes/recipe_hydrology.rst

```diff
@@ -35,6 +40,35 @@ User settings in recipe
    * start_year: 1979
    * end_year: 1979
 
+
+
+#. recipe_marrmot.yml
+
+   *Required preprocessor settings:*
+
+        *extract_shape:*
+
+            * shapefile: meuse_hydrosheds.shp (MARRMoT is a hydrological Lumped model that needs catchment-aggregated forcing data. The catchment is provided as a shapefile in Auxiliary folder in config-user.yml. The name of the shapefile is set in the recipe, here. For example, meuse_hydrosheds.shp).
+            * method: contains
+            * crop: true
+
+        *daily_statistics:*
+
+            * operator: mean (MARRMoT needs daily forcing data. Hourly forcing data are converted to daily values by mean operator).
+
+
+   *Required diagnostic settings:*
```

Comment:
The diagnostic script does not take any settings, so I think this section can be removed.

Reply:
Done.